src/api/app/controllers/webui/package_controller.rb

```diff
@@ -815,36 +817,10 @@ def set_job_status
   def live_build_log
     required_parameters :arch, :repository
 
-    if Project.exists_by_name(params[:project])
-      @project = Project.get_by_name(params[:project])
-    else
-      flash[:error] = "Couldn't find project '#{params[:project]}'. Are you sure it still exists?"
-      redirect_back(fallback_location: root_path)
-      return
-    end
-
-    begin
-      @package = Package.get_by_project_and_name(@project,
-                                                 params[:package],
-                                                 {use_source:           false,
-                                                  follow_multibuild:    true,
-                                                  follow_project_links: true})
-    rescue Package::UnknownObjectError
-      flash[:error] = "Couldn't find package '#{params[:package]}' in project '#{@project.to_param}'. Are you sure it exists?"
-      redirect_to project_show_path(@project.to_param)
-      return
-    end
-
-    if @package && !@package.check_source_access?
-      flash[:error] = 'Could not access build log'
-      redirect_to action: :show, project: @project.name, package: @package.name
-      return
-    end
-
     @build_container = params[:package] # for remote and multibuild package
-    @package ||= params[:package] # for remote case
-    @arch = params[:arch]
-    @repo = params[:repository]
+    # Make sure objects don't contain invalid chars (eg. '../')
```

Comment:
By checking that the objects exist. The backend stores data as files, eg. home:bgeuken/my_package. If the name is valid, we should be good.